static/js/tools/scatter2/plot_options.tsx

```diff
@@ -154,6 +201,51 @@ function PlotOptions(): JSX.Element {
   );
 }
 
+function getCurrentYear(): number {
+  return new Date().getFullYear();
+}
+
+function getNToM(n: number, m: number, descending = false): Array<number> {
```

Comment:
please add brief descriptions for these helpers

Reply:
fixed